uSkyBlock-Core/src/main/java/us/talabrek/ultimateskyblock/command/island/InfoCommand.java

```diff
@@ -76,7 +76,7 @@ public void run() {
                                     score.getScore(), score.getCount(),
                                     VaultHandler.getItemName(score.getBlock())));
                         }
-                        player.sendMessage(String.format("\u00a7aIsland level is %5.2f", getState().getScore()));
+                        player.sendMessage (tr(String.format("\u00a7aIsland level is %5.2f", getState().getScore())));
```

Comment:
Ye, `String.format` uses another formatting-mode than `java.text.MessageFormat` does - the latter is the one we use in GNU Gettext (i18n).